src/com/untamedears/JukeAlert/storage/JukeAlertLogger.java

```diff
@@ -992,6 +994,17 @@ public void logSnitchBucketEmpty(Snitch snitch, Player player, Location loc, Ite
         // no victim user in this event
         this.logSnitchInfo(snitch, item.getType(), loc, new Date(), LoggedAction.BUCKET_EMPTY, player.getPlayerListName(), null);
     }
+    
+    /**
+     * Logs a message that someone destroyed a cart within the snitch's field
+     * 
+     * @param snitch - the snitch that recorded this event
+     * @param player - the player that destroyed the cart
+     * @param vehicle - the vehicle destroyed
+     */
+    public void logSnitchCartDestroyed(Snitch snitch, Player player,Vehicle vehicle) {
+    	this.logSnitchInfo(snitch, null, null, new Date(),LoggedAction.CART_DESTROY,player.getPlayerListName(),vehicle.getType().toString());
```

Comment:
I wasn't sure whether the vehicle still has a location at this point, does it still exist in the world when the event gets called?